keyboards/sdrakbs/sdrakb00/readme.md

```diff
@@ -0,0 +1,56 @@
+# sdrakb00
+
+![sdrakb00](https://raw.githubusercontent.com/drabaioli/SdraKb00/refs/heads/main/img/assembled_leds_on.jpg)
+
+11 keys hot-swap macropad with rotary encoder and VIA support.
```

Comment:
```suggestion
11 keys hot-swap macropad with rotary encoder.
```

Reply:
VIA support is mentioned in many keyboard's readmes. Is there any reason for having to remove the word "VIA" from all files in my PR?

Reply:
> VIA support is mentioned in many keyboard's readmes

Doesn't mean they should be. VIA keymaps no longer exist in this repo.

Reply:
Sounds good, I'll comply.

Just out of curiosity, what's the suggested way to document VIA support in this repo? or is there a QMK online alternative to VIA that I can point people to (and that I can personally use)?

Reply:
You don't document VIA support in this repo.